src/pudl/transform/ferc1.py

```diff
@@ -2500,7 +2502,20 @@ def process_xbrl(
             )
         )
 
-    def standardize_physical_fuel_units(self, df: pd.DataFrame) -> pd.DataFrame:
+    def to_numeric(self: Self, df: pd.DataFrame) -> pd.DataFrame:
+        """Convert columns containing numeric strings to numeric types."""
+        numeric_cols = [
+            "fuel_consumed_units",
+            "fuel_cost_per_unit_burned",
+            "fuel_cost_per_unit_delivered",
+            "fuel_cost_per_mmbtu",
```

Comment:
Could be parameterized to take a set of columns to run `pd.to_numeric()` on, and the flag for how to deal with any errors.